<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recover</class>
 <widget class="QDialog" name="Recover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>660</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px;</string>
   </property>
   <widget class="PictureBox" name="btn_recover" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="recover_confirm">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>160</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 154, 252, 1);
border-radius:10px;
color:rgba(255, 255, 255, 1)</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>101</y>
      <width>660</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(206, 220, 245, 1);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="recover_tip">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>206</y>
      <width>566</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(102, 102, 102, 1)</string>
    </property>
    <property name="text">
     <string>瑞记将恢复至我的文件</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="recover_content">
    <property name="geometry">
     <rect>
      <x>246</x>
      <y>140</y>
      <width>166</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(51, 51, 51, 1)</string>
    </property>
    <property name="text">
     <string>恢复2项内容</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="recover_title">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(25,25,25,1)</string>
    </property>
    <property name="text">
     <string>恢复操作</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PictureBox</class>
   <extends>QWidget</extends>
   <header>picturebox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
